tasks.py

```diff
@@ -369,10 +369,8 @@ def migrate(c):
     print('Running InvenTree database migrations...')
     print('========================================')
 
-    manage(c, 'makemigrations')
-    manage(c, 'migrate --noinput')
-    manage(c, 'migrate --run-syncdb')
```

Comment:
Question: are you sure we do not issue the sync command anymore?

Reply:
Added it back in :)